compiler/rustc_target/src/spec/targets/powerpc64le_unknown_freebsd.rs

```diff
@@ -13,7 +13,7 @@ pub(crate) fn target() -> Target {
             description: Some("PPC64LE FreeBSD".into()),
             tier: Some(3),
             host_tools: Some(false),
-            std: Some(false),
+            std: None,
```

Comment:
```suggestion
            std: Some(true),
```
it's FreeBSD

Reply:
As https://github.com/rust-lang/rust/pull/127265 did, when "std" support status is “?” I used “None”, but is FreeBSD special?

Reply:
In fact, in powerpc64le, the [Rust package](https://www.freshports.org/lang/rust/) is available, so both `std` and `host_tools` should be `Some(true)`. On the other hand, the Rust package for powerpc and riscv does not seem to be available, so I think None (unknown) is appropriate.

Updated powerpc64le freebsd target spec.

Reply:
FreeBSD is an operating system supported by std.

Reply:
Marked all Linux/Android/BSD/Fuchsia as supporting "std" in target-spec metadata.

(I was set them to `None` because I thought there is a plan to mechanically synchronize them with platform-support.md (the status of targets with work-in-progress std support is “?” in platform-support.md, and I thought “None” in metadata would correspond to this), but if not, I think it is reasonable to set them to true since they are “targets where std support is expected to eventually be available even if currently work-in-progress”.)